hdkeychain/extendedkey_test.go

```diff
@@ -1013,3 +1014,38 @@ func TestZero(t *testing.T) {
 		}
 	}
 }
+
+func TestExtendedKey_MaximumDepth(t *testing.T) {
```

Comment:
The standard library is littered with tests with _ in the names.  It really doesn't matter.